server/routes/products.js

```diff
@@ -0,0 +1,82 @@
+const express = require('express');
+const router = express.Router();
+const Product = require('../models/product')
+
+// Get all products
+router.get('/', async (req, res) => {
+  try {
+    const products = await Product.find()
+    res.json(products)
+  } catch(err) {
+    res.status(500).json({ message: err.message })
+  }
+})
+
+// Get one product
```

Comment:
Remember to handle a `500 internal server error` here. And a `404 not found error`